bindings/python/pymongocrypt/synchronous/state_machine.py

```diff
@@ -14,6 +14,7 @@
 
```

Comment:
These imports need to continue to work so that pymongo doesn't break:
```python
try:
    from pymongocrypt.auto_encrypter import AutoEncrypter  # type:ignore[import]
    from pymongocrypt.errors import MongoCryptError  # type:ignore[import]
    from pymongocrypt.explicit_encrypter import ExplicitEncrypter  # type:ignore[import]
    from pymongocrypt.mongocrypt import MongoCryptOptions  # type:ignore[import]
    from pymongocrypt.state_machine import MongoCryptCallback  # type:ignore[import]

    _HAVE_PYMONGOCRYPT = True
except ImportError:
    _HAVE_PYMONGOCRYPT = False
    MongoCryptCallback = object
```

Reply:
Can you add a test case for this? EG:
```python
def test_pymongo_imports(self):
    from pymongocrypt.auto_encrypter import AutoEncrypter  # type:ignore[import]
    from pymongocrypt.errors import MongoCryptError  # type:ignore[import]
    from pymongocrypt.explicit_encrypter import ExplicitEncrypter  # type:ignore[import]
    from pymongocrypt.mongocrypt import MongoCryptOptions  # type:ignore[import]
    from pymongocrypt.state_machine import MongoCryptCallback  # type:ignore[import]

```